exporter/awsemfexporter/cwlog_client.go

```diff
@@ -175,9 +176,13 @@ func (client *cloudWatchLogClient) CreateStream(logGroup, streamName *string) (t
 	return "", nil
 }
 
-func newCollectorUserAgentHandler(buildInfo component.BuildInfo) request.NamedHandler {
+func newCollectorUserAgentHandler(buildInfo component.BuildInfo, logGroupName string) request.NamedHandler {
+	fn := request.MakeAddToUserAgentHandler(collectorDistribution, buildInfo.Version)
+	if strings.Contains(logGroupName, "ContainerInsights") {
```

Comment:
Should we be more specific about the pattern of the log group name and maybe check the metric namespace as well? See https://github.com/aws-observability/aws-otel-collector/blob/main/deployment-template/eks/otel-container-insights-infra.yaml#L80-L81 and https://github.com/aws-observability/aws-otel-collector/blob/main/config/eks/prometheus/config-all.yaml#L229-L230 and https://github.com/aws-observability/aws-otel-collector/blob/main/config/ecs/container-insights/otel-task-metrics-config.yaml#L181-L186

Reply:
added the regex to matches all ContainerInsight log group patterns below
```
/aws/containerinsights/<cluster-name>/prometheus
/aws/containerinsights/<eks-cluster-name>/performance
/aws/ecs/containerinsights/<ecs-cluster-name>/performance
```